apps/minaintyg/src/pages/Certificates/components/CertificateCard/CertificateCard.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { fakerFromSchema } from '@frontend/fake'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { MemoryRouter, Route, Routes } from 'react-router-dom'
+import { certificateListItemSchema } from '../../../../schema/certificateList.schema'
+import { CertificateCard } from './CertificateCard'
+
+it('Should navigate to certificate from button', async () => {
```

Comment:
Consider wrapping these tests in a describe.

Reply:
describe should only be used for when you have separated test files or for hierarchy inside a test file. This is not the case for this component. https://vitest.dev/api/#describe